proto/eth/v1/beacon_chain.proto

```diff
@@ -335,7 +335,15 @@ message ForkChoiceNode {
     // Optimistic status of the current forkchoice node.
     bool execution_optimistic = 10;
     // Execution payload block hash of the current forkchoice node.
-    bytes execution_payload = 11 [(ethereum.eth.ext.ssz_size) = "32"];
+    bytes execution_block_hash = 11 [(ethereum.eth.ext.ssz_size) = "32"];
     // Timestamp when the node was inserted
     uint64 timestamp = 12;
+    // Validity status of the node.
+    ForkChoiceNodeValidity validity = 13;
+}
+
+enum ForkChoiceNodeValidity {
```

Comment:
Do we use a similar enum elsewhere to denote optimistic status? Wonder if there's anything we could reuse

Reply:
no, we don't